model/event/subscriptions.go

```diff
@@ -266,12 +268,12 @@ func (s *Subscription) runCustomValidation() error {
 		}
 	}
 
-	if taskPercentVal, ok := s.TriggerData["task_percent_increase"]; ok {
-		taskPercent, err := strconv.ParseFloat(taskPercentVal, 32)
+	if taskPercentVal, ok := s.TriggerData[TaskPercentChangeKey]; ok {
+		taskPercent, err := strconv.ParseFloat(taskPercentVal, 64)
 		if err != nil {
 			catcher.Add(fmt.Errorf("%s must be a number", taskPercentVal))
-		} else if taskPercent < 0 {
-			catcher.Add(fmt.Errorf("%f cannot be negative", taskPercent))
+		} else if taskPercent <= 0 {
+			catcher.Add(fmt.Errorf("%f must be positive", taskPercent))
```

Comment:
this statement should be `if taskPercent < math.Nextafter(0, math.Inf(1))`. In english this says, if task percent is less than the smallest representable positive floating point number. Floating points are weird.

Furthermore, this does not guard against Nan and Positive infinity. See https://play.golang.org/p/OmhsOeCd4Xk